processing/src/test/java/org/apache/druid/collections/spatial/ImmutableRTreeTest.java

```diff
@@ -48,6 +48,7 @@
 
 /**
  */
+@SuppressWarnings("CheckReturnValue")
```

Comment:
why this change was needed?

Reply:
This line is causing this warning during strict compilation check. https://github.com/apache/druid/blob/a8eaa1e4ed81f94fe53ae14bbb078678e35de105/processing/src/test/java/org/apache/druid/collections/spatial/ImmutableRTreeTest.java#L583 Although this line is in unused tests and can be removed, thought by suppressing the check won't disturb any existing logic check that I'm not aware of.